ansible/roles/postgresql/templates/pg_hba.conf.j2

```diff
@@ -17,6 +17,9 @@ local   replication     postgres                                peer
 {% if postgres_users.get('replication') %}
 host    replication     {{ postgres_users.replication.username }}   {{ cluster_ip_range|default('0.0.0.0/0') }}    md5
 {% endif %}
+{% if postgres_users.get('devreadonly') %}
+host    replication     {{ postgres_users.devreadonly.username }}   {{ cluster_ip_range|default('0.0.0.0/0') }}    md5
```

Comment:
i wonder why we need this, are we using replication in swiss? does devreadonly need replication permissions?

Reply:
I think it's used by Datadog

Reply:
I think in the other envs (can check the vault files) the `devreadonly` user is defined in the vault file and so it gets added below on line 37: https://github.com/dimagi/commcare-cloud/pull/1288/files#diff-d959b4d4814d1bc5d1d095125136a90aR37

I think it would be better to be consistent and add it that way. The reason we have special lines for `replication` and `netvault` is that they require 'special' setting.

Reply:
yes, i wondered why devreadonly also needed "replication" permissions or be setup , i think it doesn't. and i agree, let's do it as it's done in other envs (users in the vault)